### PR TITLE
feat(contact-form): Adding examples - OEL-1055

### PR DESCRIPTION
**Pull request by planctus**

### CODE REVIEW

File: src/components/bcl-alert/__snapshots__/alert.test.js.snap [hunk 1]
```diff
@@ -14,7 +14,7 @@ exports[`OE - Alert danger renders correctly 1`] = `
       />
     </svg>
     <div
-      class="alert-content"
+      class="alert-content flex-grow-1"
```
Comment (papegaill) [inline src/components/bcl-alert/__snapshots__/alert.test.js.snap:17]: `.flex-grow-1` seems useless?

File: src/components/bcl-form/form.html.twig [hunk 2]
```diff
@@ -98,6 +117,13 @@
   {% endif %}
 {% endfor %}
 {% endif %}
+{% if _antispam is not empty %}
+  <div class="mt-4 mb-2">
+    <div class=" bg-light">
```
Comment (papegaill) [inline src/components/bcl-form/form.html.twig:122]: space to be removed before the class `.bg-light`

File: src/compositions/bcl-contact-form/contact-form.story.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        name: "Mockup - Contact form",
+        type: "figma",
+        url: "https://www.figma.com/file/NQlGvTiTXZYN8TwY2Ur5EI/BCL-Features?node-id=5191%3A180830",
+      },
+    ],
+  },
+};
+
+export const Contact = () => contactForm(correctPaths(dataDefault));
+console.log({ ...baseData, dataDefault });
```
Comment (papegaill) [inline src/compositions/bcl-contact-form/contact-form.story.js:75]: `console.log` to be removed

**Review by papegaill — CHANGES_REQUESTED** [review]

File: src/compositions/bcl-base-templates/content-type.html.twig [hunk 4]
```diff
@@ -70,6 +71,16 @@
 {% endif %}
 
 <main {{ attributes }}>
+{% if messages is defined and messages is not empty %}
+  {% block messages_area %}
+  <div class="container mt-lg-5 mt-2">
```
Comment (papegaill) [inline src/compositions/bcl-base-templates/content-type.html.twig:76]: maybe margin classes have to be set on the alert itself, when removing the alert message, the container stays empty with these spacing classes.

File: src/components/bcl-alert/__snapshots__/alert.test.js.snap (hunk 1 above)
Comment (planctus) [reply src/components/bcl-alert/__snapshots__/alert.test.js.snap:17]: This is without...it's flex, so the alert-content doesn't use all the available room if not needed.
image

File: src/compositions/bcl-base-templates/content-type.html.twig (hunk 4 above)
Comment (planctus) [reply src/compositions/bcl-base-templates/content-type.html.twig]: yep, you're right..

**Review by papegaill — APPROVED** [review]

**Pull request merged**